### PR TITLE
Update Renato's profile

### PR DESCRIPTION
**Pull request by malvikasharan**
# Summary

Update Renato's profile.

## What else?

Share this PR with others and ask them to update their profile.

### CODE REVIEW

**Comment by unode** [comment]
:+1: :+1: :+1:

File: _data/people.yaml [hunk 1]
```diff
@@ -1750,10 +1750,15 @@ unode:
     country: Germany
     pronouns: he/him
     expertise:
-        - OLS-1 project lead
+        - Full-stack developer, Computational Training, Reproducibility, OLS-1 project lead
```
Comment (bebatut) [inline _data/people.yaml:1753]:
```suggestion
        - Full-stack developer
        - Computational Training
        - Reproducibility
        - OLS-1 project lead
```

File: _data/people.yaml [hunk 2]
```diff
@@ -1750,10 +1750,15 @@ unode:
...
+    life-science-expertise: |
+        - Computational biology, Metagenomics, Meta-transcriptomics
```
Comment (bebatut) [inline _data/people.yaml:1755]:
```suggestion
        - Computational biology
        - Metagenomics
        - Meta-transcriptomics
```

File: _data/people.yaml [hunk 3]
```diff
@@ -1750,10 +1750,15 @@ unode:
...
     bio: >
```
Comment (bebatut) [inline _data/people.yaml:1756]:
```suggestion
    bio: |
```

File: _data/people.yaml [hunk 4]
```diff
@@ -1750,10 +1750,15 @@ unode:
...
-        Renato is a biologist by training with a passion for computational work.
-        Love learning about as much as teaching. Open source junky. Certified computer geek.
+        Renato is a computational biologist with a background in evolution, genomics and the microbiome.
+        Currently project and community manager of the EMBL Bio-IT project (https://bio-it.embl.de/),
```
Comment (bebatut) [inline _data/people.yaml:1758]:
```suggestion
        Currently project and community manager of the [EMBL Bio-IT project](https://bio-it.embl.de/), 
```

File: _data/people.yaml [hunk 5]
```diff
@@ -1750,10 +1750,15 @@ unode:
     country: Germany
     pronouns: he/him
     expertise:
-        - OLS-1 project lead
+        - Full-stack developer, Computational Training, Reproducibility, OLS-1 project lead
+    life-science-expertise: |
```
Comment (bebatut) [inline _data/people.yaml:1754]:
```suggestion
    life-science-expertise: 
```

**Comment by bebatut** [comment]
Thanks @malvikasharan 😄

**Comment by unode** [comment]
Thanks @bebatut and @malvikasharan :100:

**Pull request merged**